sample-operators/tomcat-operator/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This does not seems to be different from the one in the main resources. Shouldn't we remove the whole file?

Reply:
I think it's ok for an application to define it's own logging format, but of course we can leave it out too.

Reply:
I mean, if it's necessary to have an other logging format for tests then for the application, seems to be redundant. What I propose is to delete the format settings from the test resources.

Reply:
We can delete I don't mind